.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ jobs:
                 password: $DOCKERHUB_PASSWORD
             environment:
                 CMAKE_BUILD_TYPE: Release
-                FLAGS: -Wall -Wunused-parameter -Werror
+                FLAGS: -Wall -Wextra -Wno-ignored-qualifiers -Werror
```

Comment:
Is there something like a YAML variable? So we could put the basic flag in one place and not repeat them in four different places...

Reply:
It looks like alias can help if we want to reuse the FLAGS node exactly, but not if we want to extend it...
I'll do this in another PR.